docs/features/UNIFIED_CANVAS.md

```diff
@@ -0,0 +1,96 @@
+
+## Bounding Box
+* **Generation Area** - The Bounding Box serves as the area used to inform where invocation should occur. It's Width & Height can be manipulated directly via the Bounding Box tab. 
+
+## Seam Correction
+In this context, the term `seam` is used to describe the area that Invoke will process on the boundary between existing and newly created image areas during the final generation step of outpainting. This process generates a mask and then inpaints over the edge between original image and transparent regions. These controls determine how that seam will be generated. 
+
+A wider seam and a blur setting of about 1/3 of the seam have been noted as producing consistently strong results (e.g. 96 wide and 16 blur - adds up to 32 blur with both sides). Seam strength of 0.7 is best for reducing hard seams.
+* **Seam Size** - This setting controls the size of the seam masked area
+* **Seam Blur** - This setting controls the size of the blur on *each* side of the masked area
+* **Seam Strength** - This setting controls the Image2Image strength applied to the seam area
+* **Seam Steps** - This setting controls the number of steps applied to the seam.
+
+## Infill & Scaling
+* **Scale Before Processing & W/H**: When generating images with a bounding box smaller than the optimized W/H of the model (e.g., 512x512 for SD1.5), this feature first generates at a larger size with the same aspect ratio, and then scales that image down to fill the selected area. It is particularly useful when inpainting very small details. This feature is an optional feature that is enabled by default.
+* **Inpaint Replace**: When inpainting, the default method is to utilize the existing RGB values of the base layer to inform the generation process. This feature instead fills the area with noise (completely replacing the original RGB values at an Inpaint Replace value of 1), which can help generate more significant variations from the original image. You should always use a higher ImageToImage strength value when using Inpaint replace, especially at high Inpaint Replace values
+* **Infill Method**: Patchmatch and Tile are two methods for producing RGB values for use in the outpainting process. We believe that Patchmatch is a superior method, however provide support for Tile in the event that Patchmatch cannot be installed or is unavailable.
+* **Tile Size**: When using the Tile method for generation, Tile will source small portions of the original image, and randomly place these in the area being outpainted. This value sets the size of those tiles.
+
+# Hot Keys
+The Unified Canvas is a tool that excels when users utilize hotkeys. You can view the full list of keyboard shortcuts, updated with all new features, by clicking the Keyboard Shortcuts icon at the top right of the InvokeAI WebUI.
```

Comment:
You might consider adding a new section entitled "How it works." I suggest this because when you have intermediate images turned on their are at least three distinct phases of rendering and re-rendering, and some people would be curious about what's happening under the hood.

Reply:
I think these would better be detailed in the Inpainting/Outpainting docs themselves, no? The Canvas is more of an interface, whereas the "under the hood" details would be in the generation method itself